src/components/card/CardSidebarTabAttachments.vue

```diff
@@ -136,15 +154,24 @@ export default {
 		this.$store.dispatch('fetchAttachments', this.card.id)
 	},
 	methods: {
+		handleDropFiles(event) {
+			this.isDraggingOver = false
+			this.onLocalAttachmentSelected(event.dataTransfer.files[0])
+			event.dataTransfer.value = ''
+		},
+		handleUploadFile(event) {
+			this.onLocalAttachmentSelected(event.target.files[0])
+			event.target.value = ''
+		},
 		clickAddNewAttachmment() {
 			this.$refs.localAttachments.click()
 		},
-		async onLocalAttachmentSelected(event) {
+		async onLocalAttachmentSelected(file) {
```

Comment:
Please also add a check if the user has edit permissions on a board here, that is a leftover from the first attachments pr.